dll/win32/oleaut32/msvc.S

```diff
@@ -12,10 +12,62 @@
 #include <asm.inc>
 .code64
 
+EXTERN __chkstk:PROC
+
+// DWORD_PTR
+// call_method(
+//     void *func, // rcx
+//     int nb_args, // edx
+//     const DWORD_PTR *args );
 PUBLIC call_method
-call_method:
-    int 2ch
+FUNC call_method
+    push rbp
+    .PUSHREG rbp
```

Comment:
So the `.PUSHREG` MASM thing doesn't generate any executable code? (thus the `push` opcode above is needed as well?)

Reply:
https://letmegooglethat.com/?q=.PUSHREG

Reply:
That's precisely because I saw the MSDN "documentation" about it that I asked about that.